export/index.template.html

```diff
@@ -228,16 +228,23 @@ <h2>Message log</h2>
       };
 
       /**
-       * Takes a hex string (e.g. "e4567ab") and returns an array buffer (Uint8Array)
+       * Takes a hex string (e.g. "e4567abc") and returns an array buffer (Uint8Array)
        * @param {string} hexString
+       * @param {number} length: optional expected length of the resulting buffer
        * @returns {Uint8Array}
        */
-       function uint8arrayFromHexString(hexString) {
+       var uint8arrayFromHexString = function(hexString, length) {
```

Comment:
Doesn't really matter much, but any particular reason we're now declaring the function in this way? (as opposed to just `function uint8...`)